students/Dustin_L/lesson04/file_lab.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import os
+import os.path as osp
```

Comment:
although this is totally valid, I would use it sparingly

someone reading your code will not automatically know what `osp` module is and might get confused

some of the common use cases for using `as` in modules imports are:
* you are importing two modules with the same name
* you are dealing with different versions of modules and the same lib was renamed or have to use replacement similar lib
* the name of the module is really long or really confusing
* adhering to common pattern in the project or python community